lib/builder.js

```diff
@@ -141,6 +156,14 @@ function addExtraOutputs(output, tree, opts) {
 Builder.prototype.buildTree = function(tree, outFile, opts) {
   var loader = this.loader;
   var self = this;
+
+  // allow passing opts
+  // as second argument.
+  if(isPlainObject(outFile)){
```

Comment:
Even better `if (outFile && typeof outFile === 'object')` IMHO (less characters and faster). Yeah I'm happy with that, don't see why we care about prototype. An "actual" `outFile` parameter certainly wouldn't be a class instance.